lib/auth0/api/v2/tickets.rb

```diff
@@ -9,16 +9,20 @@ module Tickets
         # @see https://auth0.com/docs/api/v2#!/Tickets/post_email_verification
         # @param user_id [string] The user_id of for which the ticket is to be created.
         # @param result_url [string] The user will be redirected to this endpoint once the ticket is used.
+        # @param ttl_sec [integer] The ticket's lifetime in seconds starting from the moment of creation.
+        # After expiration, the ticket cannot be used to verify the user's email. If not specified or if
+        # you send 0, the Auth0 default lifetime of five days will be applied
         #
         # @return [json] Returns the created ticket url.
-        def post_email_verification(user_id, result_url: nil)
+        def post_email_verification(user_id, result_url: nil, ttl_sec: nil)
           if user_id.to_s.empty?
             raise Auth0::InvalidParameter, 'Must supply a valid user id to post an email verification'
           end
           path = "#{tickets_path}/email-verification"
           request_params = {
             user_id: user_id,
-            result_url: result_url
+            result_url: result_url,
+            ttl_sec: ttl_sec
```

Comment:
Since the API is looking for an integer, I think it would be worth it to check with something like:

```ruby
ttl_sec: ttl_sec if ttl_sec.is_a? Integer
```

... or similar. The API will return a `400` so we can prevent that before an HTTP call is made.